File: back-end/sistemainventario/src/main/java/co/com/uco/sistemainventario/comando/ComandoCredito.java
package co.com.uco.sistemainventario.comando;

import java.time.LocalDateTime;
import java.util.List;

public class ComandoCredito {
    private Integer idCredito;
    private Integer idCliente;
    private Integer idVendedor;
    private Integer idMoto;
    private double valorMoto;
    private double valorCuotaInicial;
    private int numeroCuotas;
    private LocalDateTime fecha;
}

File: front-end/src/app/comando/comando-credito.ts
export interface ComandoCredito {
    idCredito: number;
    idCliente: number;
    idVendedor: number;
    idMoto: number;
    valorMoto: number;
    valorCuotaInicial: number;
    numeroCuotas: number;
    valorCuota: number;
    fecha: Date;
}